# Brock Lab

## Tools

- [pycashier](https://github.com/brocklab/pycashier)

## Publication Associated Repositories

<table>
  <tr>
    <th>DOI</th>
    <th>Repo</th>
    <th>Title</th>
  </tr>
  <tr>
    <td><a href="https://doi.org/10.1371/journal.pcbi.1009104"> DOI</a></td>
    <td><a href="https://github.com/brocklab/Drug-Response-Dynamics-Model-Calibration"> Repo </a></td>
    <td>Quantification of long-term doxorubicin response dynamics in breast cancer cell lines to direct treatment schedules</td>
  </tr>
  <tr>
    <td><a href="https://doi.org/10.1088/1478-3975/abb09c"> DOI</a></td>
    <td><a href="https://github.com/brocklab/Johnson_Integrated_Modeling"> Repo </a></td>
    <td>Integrating transcriptomics and bulk time course data into a mathematical framework to describe and predict therapeutic resistance in cancer</td>
  </tr>
  <tr>
    <td><a href="https://doi.org/10.1371/journal.pbio.3000399"> DOI</a></td>
    <td><a href="https://github.com/brocklab/Johnson-AlleeGrowthModel"> Repo </a></td>
    <td>Cancer cell population growth kinetics at low densities deviate from the exponential growth model and suggest an Allee effect
    </td>
  </tr>
  <tr>
    <td><a href="https://doi.org/10.1021/acssynbio.8b00105"> DOI</a></td>
    <td><a href="https://github.com/brocklab/COLBERT_amplicon_seq"> Repo </a></td>
    <td>Control of Lineage-Specific Gene Expression by Functionalized gRNA Barcodes</td>
  </tr>
</table>
